annotations/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'p3-model-annotations'
```

Comment:
This project is named "P3-model-java". As far I remember we have two main parts here: **annotations** and **parser**. Shouldn't we rename this repository to something like "P3-model-annotations-java"? Otherwise, it will suggest that it contains also a **parser** here. 

I'm not sure if we decided already to have both in one repository - or each in a separate one.

WDYT?

Reply:
In c# we have both annotations and parser in a single repository. I think that annotations and parser are so tightly connected that there are no reason for placing them in separate repositories.

Reply:
I opt in for two modules in single repo to keep it consistent